xml/System.Xml/XmlDocumentXPathExtensions.xml

```diff
@@ -16,7 +16,7 @@
   </Base>
   <Interfaces />
   <Docs>
-    <summary>To be added.</summary>
+    <summary>Represents XPath extension methods previously found on XML document. You can use them to create XPathNavigator given XML document object.</summary>
     <remarks>To be added.</remarks>
```

Comment:
> previously found on XML document

What does this mean exactly? If a user with zero context visits MS Docs, what would they think about that statement? Does it mean it was found on XML document back in .NET Framework?

If the summary comment refers to .NET Framework, then it should go in remarks and we should make sure it clearly specifies that. But I looked for them and I couldn't find them in .NET Framework. Do you know if the methods had different names?

Extension class summaries usually also list the classes they are extending:

```suggestion
    <summary>Provides extension methods for the <see cref="T:System.Xml.XmlDocument" /> and <see cref="T:System.Xml.XmlNode" /> for document navigation.</summary>
    <remarks>To be added.</remarks>
```

Reply:
If you find the original .NET Framework methods, let me know and I can edit my suggestion remark with the proper message.

Reply:
on full framework and since .NET Core 2 those are regular methods living directly on the type, this type with extension methods is only used in .NET Core 1 where we split Xml into smaller libs